config/default/field.field.node.judgment.field_files.yml

```diff
@@ -39,7 +39,7 @@ default_value_callback: ''
 settings:
   file_directory: judgment/files
   file_extensions: 'txt doc docx xls pdf ppt pps odt ods odp rtf'
-  max_filesize: '100 MB'
+  max_filesize: '500 MB'
```

Comment:
You'll also need to update `nginx/liiweb.conf` othewise nginx won't allow large file uploads before it even gets to Drupal.

Reply:
Have adjusted nginx.conf to 500mb